recipe/meta.yaml

```diff
@@ -1,38 +1,56 @@
-{% set version = "3.0.0b2.post2" %}
+{% set version = "3.0.0b2" %}
```

Comment:
Can you compare the GitHub tarball contents to the PyPI 3.0.0b2.post2 contents to see if anything significant changed?  protobuf made a post2 release on PyPI for some reason and it would be a shame not to have whatever was fixed by this release in the conda package.